apps/spruce/src/pages/Version.tsx

```diff
@@ -229,7 +229,7 @@ export const VersionPage: React.FC = () => {
         <PageLayout>
           <PageContent>
             <VersionTabs
-              childPatches={childPatches}
+              {...(childPatches ? { childPatches } : {})}
```

Comment:
This is a little confusing to read. I think if you use the typing from below you can avoid this logic.